### PR TITLE
feat(LERN-10): added documentation

### PR DESCRIPTION
**Pull request by deepspace1000**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+echo '
+
+     /$$                                         /$$   /$$ /$$$$$$$
+    | $$                                        | $$  | $$| $$__  $$
+    | $$  /$$$$$$   /$$$$$$   /$$$$$$  /$$$$$$$ | $$  | $$| $$  \ $$
+    | $$ /$$__  $$ |____  $$ /$$__  $$| $$__  $$| $$  | $$| $$$$$$$/
+    | $$| $$$$$$$$  /$$$$$$$| $$  \__/| $$  \ $$| $$  | $$| $$____/
+    | $$| $$_____/ /$$__  $$| $$      | $$  | $$| $$  | $$| $$
+    | $$|  $$$$$$$|  $$$$$$$| $$      | $$  | $$|  $$$$$$/| $$
+    |__/ \_______/ \_______/|__/      |__/  |__/ \______/ |__/
```
Comment (A1eeee) [inline setup.sh:19]: I love it.

File: doc/adr/0000-technology-decision.md
Comment (A1eeee) [inline doc/adr/0000-technology-decision.md:1]: I think this is for the future xD

**Review by A1eeee — APPROVED** [review]
In total, this is a very good README structure for now and also a very attractive base structure, that we could use in the future to expand.

- Good Overview
- Good documentation; as an example, the documentation how to set up the back- and frontend of the project

> Only half of programming is coding. The other 90% is debugging.

File: doc/adr/0000-technology-decision.md
Comment (deepspace1000) [reply doc/adr/0000-technology-decision.md:1]: Yes its just so we already have the directory when we need it.

**Pull request merged**